### PR TITLE
Update kubernetes instructions

### PR DESCRIPTION
**Pull request by joshblakeley**
Removed outdated info around service mesh - reference new operator and add note around beta status.

### CODE REVIEW

File: tyk-docs/content/getting-started/installation/with-tyk-on-premises/with-kubernetes/with-kubernetes.md [hunk 1]
```diff
@@ -10,98 +10,30 @@ url: "/getting-started/installation/with-tyk-on-premises/with-kubernetes"
...
+## Tyk Kubernetes Operator and Ingress 
+
 {{< note success >}}
 **Note**  
 
-The helm chart installs Tyk as a "sharded" deployment, this means that all APIs that get exposed to the outside of the cluster are tagged with the `ingress` tag. If an API is not tagged, it does not get loaded and you may experience `404s`.
-You can set a tag for your exposed services in the API Designer, under the "Advanced Options" tab, the section called `Segment Tags (Node Segmentation)`, this allows you to add new tags. To make an API public, simply add `ingress` to this section, click the "Add" button, and save the API.
-
-If you are using an Ingress spec, then the Tyk K8s controller will do this for you.
+The newer Tyk Kubernetes Operator succeeds our Ingress controller and is going to be the preferred way to use Tyk with Kubernetes going forward. It is currently in BETA and we welcome users to try it out and feedback on the GitHub repository or via normal support channels for existing Tyk Customers.
```
Comment (asoorm) [inline tyk-docs/content/getting-started/installation/with-tyk-on-premises/with-kubernetes/with-kubernetes.md]: link to the operator? github.com/TykTechnologies/tyk-operator

Comment (joshblakeley) [reply tyk-docs/content/getting-started/installation/with-tyk-on-premises/with-kubernetes/with-kubernetes.md]: Would be a good start, wouldnt it!

File: tyk-docs/content/getting-started/installation/with-tyk-on-premises/with-kubernetes/with-kubernetes.md [hunk 2]
```diff
@@ -10,98 +10,30 @@ url: "/getting-started/installation/with-tyk-on-premises/with-kubernetes"
...
+## Tyk Kubernetes Operator and Ingress 
+
 {{< note success >}}
 **Note**  
 
-The helm chart installs Tyk as a "sharded" deployment, this means that all APIs that get exposed to the outside of the cluster are tagged with the `ingress` tag. If an API is not tagged, it does not get loaded and you may experience `404s`.
-You can set a tag for your exposed services in the API Designer, under the "Advanced Options" tab, the section called `Segment Tags (Node Segmentation)`, this allows you to add new tags. To make an API public, simply add `ingress` to this section, click the "Add" button, and save the API.
-
-If you are using an Ingress spec, then the Tyk K8s controller will do this for you.
+The new Tyk Kubernetes Operator succeeds our Ingress controller and is going to be the preferred way to use Tyk with Kubernetes going forward. It is currently in BETA and we welcome users to try it out and feedback on the GitHub repository or via normal support channels for existing Tyk Customers.
```
Comment (asoorm) [inline tyk-docs/content/getting-started/installation/with-tyk-on-premises/with-kubernetes/with-kubernetes.md:26]: `BETA` -> `ALPHA`.

**Review by sedkis — APPROVED** [review]

File: tyk-docs/content/getting-started/installation/with-tyk-on-premises/with-kubernetes/with-kubernetes.md (hunk 2 above)
Comment (joshblakeley) [reply tyk-docs/content/getting-started/installation/with-tyk-on-premises/with-kubernetes/with-kubernetes.md:26]: "Early beta" ? ;)

**Comment by joshblakeley** [comment]
Ready for merge please @marksou

**Pull request merged**